merkle-tree/indexed/src/lib.rs

```diff
@@ -229,4 +232,35 @@ where
 
         Ok(())
     }
+
+    /// Initializes the address merkle tree with the given initial value.
+    /// The initial value should be a high value since one needs to prove non-inclusion of an address prior to insertion.
+    /// Thus, addresses with higher values than the initial value cannot be inserted.
+    pub fn initialize_address_merkle_tree(
+        address_merkle_tree_inited: &mut IndexedMerkleTree<'a, H, I, HEIGHT>,
+        init_value: BigUint,
+    ) -> Result<(), IndexedMerkleTreeError> {
+        let mut indexed_array = IndexedArray::<H, I, 2>::default();
```

Comment:
That's not exactly what I was thinking about.

First of all, we already have the `init` method in line 138-141. Can't we just do it there? We are appending 0 exactly in that place, so I think we could add the highest element there as well?

Reply:
Nevermind, I guess we don't want to add the `MODULUS - 1` element to nullifier queues. I'm fine with keeping it as a separate method.

Reply:
added it to the init function of the indexed Merkle tree not indexed array so it's independent from nullifier queues.
Also we switched from indexed arrays to hash sets everywhere anyway, or are we still using indexed arrays somewhere onchain?

Reply:
>Also we switched from indexed arrays to hash sets everywhere anyway, or are we still using indexed arrays somewhere onchain?

I don't think we use indexed arrays on-chain. We should probably kick them out to a separate crate and make sure we don't use it onchain, only in tests and WASM modules.